burst/provider.py

```diff
@@ -242,6 +251,19 @@ def process(provider, generator, filtering, verify_name=True, verify_size=True):
                         csrf_token = re.search(r'name="csrfToken" value="(.*?)"', client.content)
                         url_search = url_search.replace("CSRF_TOKEN", csrf_token.group(1))
 
+        if provider == 'lostfilm':
+            log.info('[%s] Need open page before search', provider)
+            url_search=url_search.replace('marvels_','')
```

Comment:
They have too many differences between original titles and on-site titles. 
So making replaces in the code is a bad idea. 
If you change i will accept the pull.